content/en/docs/reference/glossary/dra.md

```diff
@@ -0,0 +1,24 @@
+---
+title: Dynamic Resource Allocation
+id: dra
+date: 2025-05-13
+full_link: /docs/concepts/scheduling-eviction/dynamic-resource-allocation/
+short_description: >
+  A Kubernetes feature for requesting and sharing resources among Pods. These
+  resources are often attached devices like hardware accelerators.
+
+aka: DRA
```

Comment:
Is this the right format for the `aka` field? This is rendering as `Also known as: 68, 82, 65` (ASCII for D, R, A).

A cursory look suggests this should be an array:
```suggestion
aka:
- DRA
```